fwoa-tools/src/export-script.py

```diff
@@ -230,7 +230,7 @@ def additional_transformations(record):
 
 
 def add_dup_resource_type(record):
-    record["partitionKeyDup"] = record["resourceType"]
+    record["partitionKeyDup"] = record["resourceType"] + "-v" + record["meta"]["versionId"]
```

Comment:
Will we be adding the padding zeros here or in another place?

Reply:
We don't need the the `0` because we're sorting it in the `migrationImport.ts` script.